src/__a11y__/to-validate-a11y.ts

```diff
@@ -58,10 +78,10 @@ async function toValidateA11y(this: jest.MatcherUtils, element: HTMLElement) {
     const flattenAxeViolations = flatMap(axeViolations, violation =>
       violation.nodes.map(node => `${node.failureSummary} [${violation.id}]`)
     );
-    const allViolations = uniq([...htmlViolations, ...flattenAxeViolations]).map(
-      (message, index) => `${index + 1}. ${message}`
-    );
-    return ['Expected HTML to be valid but had the following errors:'].concat(allViolations).join('\n');
+
+    return ['Expected HTML to be valid but had the following errors:']
+      .concat(formatMessages('HTML validation', htmlViolations), formatMessages('Axe checks', flattenAxeViolations))
```

Comment:
I was very confused at first and it took significant time until I figured that the errors are not from axe, but from html linter. 

Maybe the new message formatting will help